File: lab_danger/mail.py
import smtplib
from email.mime.text import MIMEText

class sendGmail:
    username, password = 'sist.kudolab@gmail.com', 'kudo0401'

    def __init__(self, to, sub, body):
        host, port = 'smtp.gmail.com', 465
        msg = MIMEText(body)
        msg['Subject'] = sub
        msg['From'] = self.username
        msg['To'] = to

        smtp = smtplib.SMTP_SSL(host, port)
        smtp.ehlo()
        smtp.login(self.username, self.password)
        smtp.mail(self.username)
        smtp.rcpt(to)
        smtp.data(msg.as_string())
        smtp.quit()

to = 'koyamaryoma@gmail.com'
sub = 'python smtplib'
body = 'hello world'
sendGmail(to, sub, body)

to = 'yukisnow0704@gmail.com'
sub = 'python smtplib'
body = 'hello world'
sendGmail(to, sub, body)

File: lab_danger/mail.php
<?php
require_once ( 'PHPMailerAutoload.php' );
$subject = "タイトル";
$body = "メール本文";
$fromname = "me";
$from = "sist.kudolab@gmail.com";
$smtp_user = "sist.kudolab@gmail.com";
$smtp_password = "kudo0401";

$mail = new PHPMailer();
$mail->IsSMTP();
$mail->SMTPDebug = 1; // debugging: 1 = errors and messages, 2 = messages only
$mail->SMTPAuth = true;
$mail->CharSet = 'utf-8';
$mail->SMTPSecure = 'tls';
$mail->Host = "ssl:smtp.gmail.com";
$mail->Port = 465;
$mail->IsHTML(false);
$mail->Username = $smtp_user;
$mail->Password = $smtp_password; 
$mail->SetFrom($smtp_user);
$mail->From     = $from;
$mail->Subject = $subject;
$mail->Body = $body;
$mail->AddAddress('yukisnow0704@gmail.com');

if( !$mail -> Send() ){
    $message  = "Message was not sent<br/ >";
    $message .= "Mailer Error: " . $mailer->ErrorInfo;
} else {
    $message  = "Message has been sent";
}

echo $message;